src/Components/ServiceScene/Breakdowns/SelectLabelActionScene.tsx

```diff
@@ -299,6 +300,9 @@ export class SelectLabelActionScene extends SceneObjectBase<SelectLabelActionSce
 
   public clearFilters = (variableType: InterpolatedFilterType) => {
```

Comment:
It'd be nice to add a regression so this doesn't happen again, and is one less thing to manually QA.